Cargo.toml

```diff
@@ -29,13 +29,15 @@ keywords = ["gamedev", "cgmath", "collision"]
 [lib]
 name = "collision"
 
-[features]
-eders = ["serde", "serde_derive", "cgmath/serde"]
-
 [dependencies]
 num = "0.1"
 rand = "0.3"
 approx = "0.1"
-cgmath = "0.15"
+serde_derive = "1.0"
 serde = { version = "1.0", optional = true }
```

Comment:
should just do `features = ["serde_derive"]` here and avoid pulling `serde_derive` manually

Reply:
I'm sorry I'm not sure I follow.  I don't think the serde crate has a `serde_derive` feature.

Reply:
see https://github.com/brendanzab/cgmath/blob/3543ac02745c43091d16560a63ca1c346c10bd1f/Cargo.toml#L27

Reply:
Huh!  Well how about that, it does.  Well that's awesome, thanks!  I'll do that!